.env.example

```diff
@@ -9,4 +9,10 @@ USER_SECRET=
 
 PORT=
 SEED=
-CHAIN_ID=
\ No newline at end of file
+CHAIN_ID=
+
+
+
+CONTRACT_CHAT_ADDRESS=
+RPC_SERVER=
+MY_ADDRESS=
```

Comment:
My PR changes these variable names, hehe

Reply:
that is fine 😺